app/controllers/api/v4/imports_controller.rb

```diff
@@ -1,5 +1,7 @@
 class Api::V4::ImportsController < ApplicationController
   skip_before_action :verify_authenticity_token
+  before_action :doorkeeper_authorize!
```

Comment:
This is a helper that `doorkeeper` provides us. It carries out the token verification based on our configs.